src/sources/soundsource.cpp

```diff
@@ -32,7 +32,7 @@ QString SoundSource::getTypeFromUrl(const QUrl& url) {
 
 //static
 QString SoundSource::getTypeFromFile(const QFileInfo& fileInfo) {
-    const QString fileSuffix = fileInfo.suffix().toLower().trimmed();
+    const QString fileSuffix = fileInfo.completeSuffix().toLower().trimmed();
```

Comment:
This would change the file type recognition for files names like `artistname.songtitle.flac`
I'm not sure if this case is relevant, but `fileinfo.filename().toLower().trimmed().endsWith(".stem.mp4")` below would be the safe way to implement this.